src/components/Settings/ChangePassword.vue

```diff
@@ -58,6 +58,9 @@ export default {
     hasOldPassword () {
       return !!this.oldPassword
     },
+    hasPasswordChanged () {
+      return !(this.oldPassword === this.newPassword)
```

Comment:
Looks good.
One minor thing, though, regarding code style: You can simplify 

```code
return !(this.oldPassword === this.newPassword)
```
to
```code
return this.oldPassword !== this.newPassword
```

Reply:
i will update the changes @mr-kojo  :+1: